packages/terra-arrange/docs/Arrange.md

```diff
@@ -0,0 +1,9 @@
+| Prop Name | Type | Is Required | Default Value | Description |
```

Comment:
Decided to just generate when we publish. And import them into terra-ui.com site.